pallet/account-migration/src/lib.rs

```diff
@@ -480,7 +480,6 @@ pub mod pallet {
 				let now = <frame_system::Pallet<T>>::block_number();
 
 				l.unstaking_ring.retain(|(_, t)| t > &now);
-				l.unstaking_kton.retain(|(_, t)| t > &now);
```

Comment:
KTON need to be transferred back to user

Reply:
The kton has been transfered to the user in the steps before this.